modules/kubernetes-addons/main.tf

```diff
@@ -301,6 +301,18 @@ module "kube_prometheus_stack" {
   addon_context = local.addon_context
 }
 
+module "portworx" {
+  count                = var.enable_portworx ? 1 : 0
+  source               = "portworx/portworx-addon/eksblueprints"
+  version              = "0.0.3"
+  helm_config          = var.portworx_helm_config
+  manage_via_gitops    = var.argocd_manage_add_ons
```

Comment:
If you are enabling gitops support via argocd then another PR needs to be opened in this repo https://github.com/aws-samples/eks-blueprints-add-ons.

Reply:
No we are not going to support it in this release, I have removed this part